packages/algob/README.md

```diff
@@ -2,13 +2,29 @@
 
 Algob is the CLI (Command Line Interface) for Algo Builder. Think about it as an Ethereum Truffle but for Algorand projects.
 
+## Packages
```

Comment:
please remove this part, we are in packages folder which has sub-section `algob` and `runtime`. they both have separate readme

Reply:
okay